api/core/text2sql_sync.py

```
@@ -0,0 +1,635 @@
+
+        healing_result = healer_agent.heal_and_execute(
+            initial_sql=sql_query,
+            initial_error=str(exec_error),
+            execute_sql_func=execute_sql,
+            db_description=context.db_description,
+            question=question,
+            database_type=context.db_type
+        )
+
+        if not healing_result.get("success"):
+            raise exec_error
```

Ruff (0.14.14)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)